{
  "entity": {
    "location": {
      "namespaceTitle": "Uno.Threading",
      "namespaceUri": "uno/threading",
      "packageName": "Uno.Threading",
      "packageVersion": "1.8.0-rc1"
    },
    "parameters": [
      {
        "name": "reason",
        "href": "uno/exception",
        "title": "Exception",
        "fullyQualifiedTitle": "Uno.Exception"
      }
    ],
    "id": {
      "id": "uno/threading/promise_1/reject_533b340b",
      "parentId": "uno/threading/promise_1",
      "type": "Method",
      "modifiers": [
        "public"
      ]
    },
    "uri": {
      "idUri": "uno/threading/promise_1/reject_533b340b",
      "href": "uno/threading/promise_1/reject_533b340b"
    },
    "titles": {
      "pageTitle": "Promise.Reject Method (Exception)",
      "navigationTitle": "Reject Method (Exception)",
      "fullTitle": "Uno.Threading.Promise`1.Reject",
      "indexTitle": "Reject(Exception)",
      "fullyQualifiedIndexTitle": "Uno.Threading.Promise.Reject(Uno.Exception)"
    },
    "syntax": {
      "uno": "public voidReject(Uno.Exception reason) {}"
    }
  },
  "tableOfContents": {}
}